### ISSUE TITLE
Camera Movement

### ISSUE THREAD

**Issue by HazilMohamed** [viromedia/viro#707]
Is there any way to move camera around, so I can move my character around? I can't find it on the docs

**Comment by radvani** [comment]
Yes please see our camera guide [here](https://docs.viromedia.com/docs/camera).
